src/components/TableBodyCell.js

```diff
@@ -134,7 +134,7 @@ function TableBodyCell(props) {
         className,
       )}
       {...otherProps}>
-      {children}
+      {typeof children === 'function' ? children(dataIndex, rowIndex) : children}
```

Comment:
Maybe children could be a function.
I thought it was only a react node(element)

Does children mean columns?

Reply:
Yeah, it's a pattern called "Function as Child Component". It's useful in certain situations. Here the children equal either the value that will be displayed in the column, or the customBodyRenderLite function, which will generate the value for the column.

Reply:
@patorjk  thanks!